src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -98,19 +98,19 @@ No matter which option you use to deploy Browser, the end result is to inject th
 
 Use this procedure to automatically deploy the Browser agent on an app that is monitored by APM:
 
-1. Go to **[one.newrelic.com](https://one.newrelic.com) > Browser > <Icon name="fe-plus-circle"/>
-   + Add More Data**.
-2. Find the header "Back-end, front-end, and mobile applications" and select the New Relic Browser tile. A menu with account options will appear and prompt you to select the account you want to add this instrumentation to. Choose your account and click the continue button.
-4. From the Browser deployment page, select **Enable via APM**.
-5. Select your choice of available options for **Browser agent instrumentation** [features](/docs/browser/new-relic-browser/getting-started/new-relic-browser).
-6. Type or use the search window to find the app's name.
-7. Select **Enable**.
+1. Go to **[one.newrelic.com](https://one.newrelic.com)**, select **Browser**, and then select **Add more data**.
+2. In the **Back-end, front-end, and mobile applications** section, select the **New Relic Browser** tile. When prompted to select the account you want to add this instrumentation to, choose your account, and click **Continue**.
+4. From the **Get started with New Relic Browser page**, in the **Choose a deployment method** section, select **Enable via New Relic APM**.
+5. In the **Choose your instrumentation** section, select **Lite**, **Pro**, or **Pro + SPA**. See [features](/docs/browser/new-relic-browser/getting-started/new-relic-browser) for details about each .
```

Comment:
I'm not sure that the link correctly supports the selections in this section. If you're okay with the changes, but the link is wrong, let's find a better one.